man/memkind_memtier.3

```diff
@@ -53,6 +53,8 @@ functionality is considered as stable API (STANDARD API).
 .BI "size_t memtier_usable_size(void " "*ptr" );
 .br
 .BI "void memtier_free(void " "*ptr" );
+.br
+.BI "size_t memtier_tier_allocated_size(struct memtier_tier  " "*tier");
```

Comment:
nit: Remove extra whitespace after `memtier_tier` and add a whitespace before closing brackets.